akka-stream/src/main/scala/akka/stream/impl/ActorFlowMaterializerImpl.scala

```diff
@@ -74,7 +74,7 @@ private[akka] case class ActorFlowMaterializerImpl(override val settings: ActorF
             mat
 
           case stage: StageModule ⇒
-            val (processor, mat) = processorFor(stage, effectiveAttributes, calcSettings(effectiveAttributes)(settings))
+            val (processor, mat) = processorFor(stage, effectiveAttributes.and(OperationAttributes.materializationName(flowName)), calcSettings(effectiveAttributes)(settings))
```

Comment:
It only contains names of stages, not the entire `flowName`. And that's the one I need in order to show to be able to correlate log statements for a given materialization. If one materialization fails and another one doesn't, you want to see their logs separately - which is why we need to log "flow-1544" / "flow-2323" etc.

flowName is: `val name = s"$flowName-$nextId-${attr.name}"`, so `flow-1`